targets/bcm63xx-generic

```diff
@@ -0,0 +1,12 @@
+include 'bcm63xx.inc'
+
+device('tp-link-td-w8968-v3', 'sagem_fast-2704n', {
+	sysupgrade = '-squashfs-cfe', 
+	sysupgrade_ext = '.bin', -- No sysupgrade image available
+
+	manifest_aliases = {
+		'FAST2704N', -- Upgrade from OpenWrt 19.07
+	},
+
```

Comment:
```suggestion
aliases = {'tp-link-td-w8968-v3'},
```
And fixing the tp-link reference in line three to the sagem reference you found in your tests should do the trick.